### ISSUE TITLE
A one-off transfer to a reward account should be rejected

### ISSUE THREAD

**Issue by edd** [vegaprotocol/vega#6066]
tldr a one-off transfer does not support distribution metric and markets in scope and therefore should not be used for reward account type that requires it or else the transferred funds will be locked an inaccessible. While it's not something someone should do (a docs update will clarify this, https://github.com/vegaprotocol/documentation/issues/231), it's possible to reject it reliably so let's update the core to do that

# Feature Overview

**In order to** prevent users from trying to create one off transfers to inaccessible rewards accounts
**We will** reject any one off transfer to a reward pool that is one that requires a distribution metric 
and markets-in-scope. (https://github.com/vegaprotocol/specs/blob/master/protocol/0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts)
**So that** assets will not be moved and become untransferable

## Specs
- https://github.com/vegaprotocol/specs/blob/master/protocol/0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts

# Acceptance Criteria
A list of criteria (aim for 3!) that have to be met for this feature to be accepted as 'done' by the product owner.

- A one-off transfer to any of the reward account types that are meant to be auto-distributed to markets is rejected (`AccountType_ACCOUNT_TYPE_REWARD_LP_RECEIVED_FEES`, `AccountType_ACCOUNT_TYPE_REWARD_MAKER_RECEIVED_FEES`, `AccountType_ACCOUNT_TYPE_REWARD_TAKER_PAID_FEES`, `AccountType_ACCOUNT_TYPE_REWARD_MARKET_PROPOSERS`)

**Comment by edd** [comment]
Marked as blocked until there's a consistent view on if this should be possible, and if so what happens. 

This ticket and the spec ticket will enter the arena, and only one will emerge victorious. The other: `wontfix`.
